filter/etc/rspamd/scores.d/nethserver_group.conf

```diff
@@ -1,11 +1,14 @@
 #
 # This file will be overwritten by the next rpm update
 # use /etc/rspamd/local.d/nethserver_group.conf' - to override the defaults
-#
+# 
```

Comment:
Please, try to avoid useless modifications ;)

Reply:
done